index.html

```diff
@@ -4853,6 +4853,73 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to be usable to identify the <a>holder</a> to the <a>verifier</a>.
+        </p>
+        <p>
+Relevant metadata about the <code>holder</code> <a>property</a> is expected
+to be available to the <a>verifier</a>. For example, a <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is relevant when
+checking proofs on <a>verifiable presentations</a>.
+        </p>
+        <p>
+See the <a data-cite="VC-IMP-GUIDE/#subject-holder-relationships"></a> and
+<a data-cite="VC-USE-CASES#user-tasks"></a> for additional examples related to <a>subject</a> and <a>holder</a>.
+        </p>
+
+        <p class="note">
+`Issuer`, `subject`, and `holder` are graph nodes which support multiple representations, 
+potentially making it complex to evaluate whether these roles are being filled
+by appropriate entities.  Validation is the process by which verifiers apply business rules to 
+evaluate the appropriateness of a particular use of a Verifiable Credential.
+        </p>
+        <ul>
+          <li>
+<a href="#issuer">Issuers</a> define expressions of an `issuer` in <a>credentials</a>
+          </li>
+          <li>
+<a href="#presentations-0">Presentations</a> define expressions of a `holder` in <a>presentations</a>
+          </li>
+          <li>
+<a href="#credential-subject">Credential subjects</a> define expressions of a `credentialSubject` in <a>credentials</a>
```

Comment:
Essentially the same thing you proposed, and I accepted here: https://github.com/w3c/vc-data-model/pull/1199#discussion_r1283406163

For clarity:

JSON Members -> RDF Graph -> Knowledge Representation

"holder" -> "graph node" -> entity that possesses bytes.
"issuer" -> "graph node" -> entity that signs bytes.
"verifier" -> "graph node" -> entity that verifies bytes.
"subject" -> "graph node" -> entity about which bytes representation of claims are made.

Reply:
Unfortunately, https://github.com/w3c/vc-data-model/pull/1199#discussion_r1283406163 no longer goes there, because that discussion was resolved, and your "for clarity" segment brings me (and perhaps others) no closer to understanding what is meant by the three "define expressions" sentences.

Reply:
I agree these three sentences are confusing ... I don't really know what they mean. Some of that confusion may come from what seems to be a conflation between roles and the expression of information about the entities that play those roles. I don't believe "Presentations" or "Credential subjects" "define" anything either.